mobile/src/bundles/common/components/modal/modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Modal as RNModal } from 'react-native';
+
+import {
+    Button,
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+import { ButtonType, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    title: string;
+    visible: boolean;
```

Comment:
boolean variables supposte to start from is, are, have, had and so on

Reply:
This is the Modal's property